### PR TITLE
DM-34989: Update obs_lsst to fix common problems.

### PR DESCRIPTION
**Pull request by czwa**
* Set size of ITL overscan region to 64 pixels.
* Remove DETSEC log warning that will always trigger due to external value mismatches.
* Set the overscan.fitType default to MEDIAN_PER_ROW.

### CODE REVIEW

File: python/lsst/obs/lsst/assembly.py [hunk 1]
```diff
@@ -168,8 +168,11 @@ def logCmd(*args):
     datasec = bboxFromIraf(d["DATASEC"]) if "DATASEC" in d else None
     biassec = bboxFromIraf(d["BIASSEC"]) if "BIASSEC" in d else None
 
-    if detsec and outAmp.getBBox() != detsec:
-        logCmd("DETSEC doesn't match (%s != %s)", outAmp.getBBox(), detsec)
+    # 2022-11-11: There is a known issue that the header DETSEC does
+    # not match the cameraGeom value: DM-36115.  Disabling this test
+    # to remove useless warnings until that is resolved.
+    # if detsec and outAmp.getBBox() != detsec:
+    #     logCmd("DETSEC doesn't match (%s != %s)", outAmp.getBBox(), detsec)
```
Comment (erykoff) [inline python/lsst/obs/lsst/assembly.py:175]: I do not understand this comment.  Or rather, is this still necessary with the update to the `cameraHeader.yaml`?  Or is this about past images?

Comment (czwa) [reply python/lsst/obs/lsst/assembly.py:175]: This is all images.  The `DETSEC` recorded in the headers swaps the y-position of the upper and lower sets of amplifiers.  Craig has the exact values on the referenced ticket:
https://jira.lsstcorp.org/browse/DM-36115?focusedCommentId=856790&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-856790

I don't think it's expected to be fixed in the near future, so I removed the test to keep the log noise down to prevent confusion.

Comment (erykoff) [reply python/lsst/obs/lsst/assembly.py:175]: Okay, can you rephrase the comment so that it's clear that the header DETSEC is incorrect?  This comment and the linked issue title are ambiguous about whether the problem is the header or cameraGeom.

**Review by erykoff — APPROVED** [review]
See my suggested clarification of the code comment.
